whois-scheduler/src/main/resources/applicationContext-scheduler.xml

```diff
@@ -13,6 +13,20 @@
     <import resource="applicationContext-api.xml"/>
     <context:component-scan base-package="net.ripe.db.whois.scheduler"/>
 
-    <task:scheduler id="taskExecutor" pool-size="10"/>
-    <task:annotation-driven scheduler="taskExecutor"/>
+    <bean id="scheduler" class="net.javacrumbs.shedlock.spring.SpringLockableTaskSchedulerFactoryBean">
+        <constructor-arg><task:scheduler id="sch" pool-size="10"/></constructor-arg>
+        <constructor-arg>
+            <bean class="net.javacrumbs.shedlock.provider.jdbctemplate.JdbcTemplateLockProvider">
+                <constructor-arg ref="internalsDataSource"/>
+            </bean>
+        </constructor-arg>
+        <constructor-arg name="defaultLockAtMostFor">
```

Comment:
Add a comment to explain that SchedulerLock jobs will lock for 23 hours by default ?

Reply:
Done